tool/pom.xml

```diff
@@ -286,10 +286,12 @@
         <dependency>
             <groupId>org.jboss.logmanager</groupId>
             <artifactId>jboss-logmanager</artifactId>
+            <scope>provided</scope>
```

Comment:
Scopes are always inherited from parent pom (in this case org.wildfly.security:wildfly-elytron-parent) so this change is not necessary.

Reply:
Thanks @ropalka. When I was initially working on this, `wildfly-elytron-tool` was a shaded JAR and I had found that this change was needed to prevent `wildfly-elytron-tool` from getting transformed.

We have since updated things so `wildfly-elytron-tool` is no longer a shaded JAR so I can remove this commit now. Thanks for catching this.